typescript/packages/subsurface-viewer/src/layers/wells/wellsLayer.ts

```diff
@@ -647,6 +647,10 @@ export default class WellsLayer extends CompositeLayer<WellsLayerProps> {
     getPickingInfo({ info }: { info: PickingInfo }): WellsPickInfo {
         if (!info.object) return { ...info, properties: [], logName: "" };
 
+        if (info.sourceLayer?.constructor === WellLabelLayer) {
+            return { ...info, properties: [], logName: "" };
+        }
```

Comment:
Since the returned value is duplicate, suggest to either combine the two clauses or the returned value in order to remove duplication.